app/(frontend)/privacy-policy/page.module.scss

```diff
@@ -0,0 +1,59 @@
+.page {
```

Comment:
No need to manually style, just add [tailwind typography](https://github.com/tailwindlabs/tailwindcss-typography) and add the `prose` class. It will be more consistent and there if we need to add more pages in the future.

Reply:
Thanks. Didn't know the use case for this.